tests/beta/test_default_tensor.py

```diff
@@ -1,1831 +0,0 @@
-# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
It seems that this file is being removed :thinking:

Reply:
Yeah I'll put it back, I just removed it because it gave me trouble when running the tests on my computer.